src/ucp/proto/proto.h

```diff
@@ -42,6 +42,16 @@ typedef unsigned ucp_proto_id_t;
 typedef uint64_t ucp_proto_id_mask_t;
 
 
+/**
+ * Protocol tag for internal usage, to allow searching for specific protocols
+ */
+typedef enum {
+    UCP_PROTO_TAG_NONE,
+    UCP_PROTO_TAG_AM_SHORT,   /* The protocol uses only uct_ep_am_short() */
+    UCP_PROTO_TAG_PUT_SHORT   /* The protocol uses only uct_ep_put_short() */
+} ucp_proto_tag_t;
```

Comment:
id' not use `tag` in the name - it is confusing, like it is related to TAG API only.
maybe `ucp_proto_id_t`?

Reply:
yes, agree
id is also not suitable because it does not have to be unique
maybe "proto_flag_t" ?

Reply:
do you mean it can be a combination of, say, `BCOPY` and `MULTI`?
Then `proto_flag_t` is much better name.

Reply:
yes, make this a combination instead of enum, probably can be useful in the future